2 data.sql

```diff
@@ -0,0 +1,25 @@
+use FlyAirDB
+go 
+delete Flyair 
+go 
+
+set dateformat dmy
+insert Flyair (FlightNumber, DepartureAirport, DepartureCountry, ArrivalAirport, ArrivalCountry, DateDeparting, TimeDeparting, DateArriving, TimeArriving, PassengerFirstName, PassengerLastName, DOB, PassengerAdress, PassportNumber, PassportIssueDate, PassportExpiryDate, Nationality)
+
+select          'FLY001', 'LHR', 'UK', 'JFK', 'USA',    '10/12/2021' ,'8:00', '10/12/2021' ,'12:00', 'John' , 'Major', '31/12/2000', '22 Queens Road', '175478100', '10/09/2016', '10/09/2026', 'UK'
+union select    'FLY002', 'JFK', 'USA', 'LHR', 'UK',    '31/12/2021' , '15:00', '01/01/2022' , '14:30', 'John' , 'Major', '31/12/2000', '22 Queens Road', null , null , null, null
+union select    'FLY001', 'LHR', 'UK', 'JFK', 'USA',    '12/01/2020' , '8:00', '12/01/2020' , '12:00', 'Jill' , 'Johnson', '12/03/1997', '10 Downing Street', '367654998', '30/01/2019', '25/03/2029', 'USA'
+union select    'FLY003', 'LGW', 'UK', 'TLV', 'Israel', '20/01/2022' , '19:55', '21/01/2022' , '3:00', 'Mike' , 'Markowicz', '09/12/1970', '45 Baker Street' , null , null , null, null
+union select    'FLY004', 'TLV', 'Israel', 'LGW', 'UK', '09/10/2020' , '6:40', '09/10/2020' , '9:30', 'Nick' , 'Smith', '27/03/1998', '29 London Avenue' , null , null , null, null
+--SH bad data. John Smith is under 16. According to the spec all passenger must be above 16.   
+--union select    'FLY004', 'TLV', 'Israel', 'LGW', 'UK', '09/10/2020' , '6:40', '09/10/2020' , '9:30', 'John' , 'Smith', '20/10/2004', '29 London Avenue', '209989911', '10/09/2018', '10/09/2023', 'UK'
```

Comment:
The DOB is 20/10/2004 which works out that this person is now 19. It seems like something is wrong with your constraint. Please fix. (I don't think it's an issue to use year function...)

Reply:
The way I understood the spec, the passenger must be over 16 by departure. When calculating the difference between the DOB (20/10/2004) and DateDeparting (09/10/2020), they are 11 days missing till the age of 16?

Reply:
Your understanding is correct. The DOB is 20/10/2004, and the departure date is 09/10/2020. This makes the passenger 15 years and 354 days old on the departure date, 11 days short of turning 16.

Other students might have used only the year difference, which incorrectly shows the passenger as 16. You can leave this record commented out in the insert statement.